README.md
# Surviv.io SimpleSurviv Resource Pack

## Installation
**How to install:**

1. Download the <rules>.json file
3. Download "resource override" from the Chrome Web Store
4. Once you have gotten the extension, click on the "Resource Override" icon 
5. On the top right, there is a button labeled "Options"; click on it 
6. Click on the "Load Rules" button and select the customsoundrules.json file you should have downloaded
7. Open Surviv.io
8. Should be working

## Disclaim

*All stuff used in this extension is, unless told and credited, in the public domain!*

*This script doesn't contain any hackusations, or code that gives you a unfair advantage!
This script is used through resource override, if the game patches it, deal with it, it's not a vanilla script!*

## Help and support
*https://discordapp.com/invite/AHZYHxV*

*https://www.youtube.com/c/FilipKZ*

## Credits

Enhanced Sounds: https://github.com/FilipKZhong/Surviv.io-Sounds

HUD: https://github.com/FilipKZhong/Surviv.io-Enhanced


CodeAssets/changelog.md
# Changelog

## [Unreleased]

*   Better event mode support.
*   Custom/private games.

## [0.8.81] - December 15, 2019

*   Added new perks: Inspiration, Last Breath.
*   Added new roles to faction mode: bugler, grenadier, recon.
*   Added new weapon: bugle.

## [0.8.8] - December 2, 2019

*   Added new perks: Hollow-points, Master Scavenger.
*   Added new classes: Scout, Sniper, Medic, Demo, Assault, Tank (cobalt map only).
*   Added new building: twins bunker (cobalt map only).
*   Added new obstacles: class pods, synthetic tree, synthetic bushes, synthetic stones (cobalt map only).
*   Changed DEagle 50 world image.

### Balance

*   One in the Chamber no longer works with shotguns.

## [0.8.73] - November 26, 2019

*   Added new perk: Perky Shoot.
*   Added new outfit: Fowl Facade.
*   Added new XP artifact: Bone of Gordon.

## [0.8.72] - November 12, 2019

*   Added new perks: Flak Jacket, Explosive Rounds.
*   Fixed an issue where stats would not record when using team lobbies.

## [0.8.71] - November 5, 2019

*   Weapons in 50v50 air drops now come packaged with their specific ammo.
*   Medic in 50v50 has an additional perk: Revivify.

### Balance

*   Two "lone" survivrs will now be assigned per team in 50v50 (up from one).

## [0.8.7] - October 26, 2019

*   Added new perks: One With Nothing, Feedership, Dev Troll Special, Gabby Ghost, That Sucks, Candy Corn, Red Jelly Beans, Sour Apple Belt, Blueberry Taffy, Full Size OKAMI Bar.
*   Added new loot type: XP artifact. Looting XP artifacts grants progress towards the Survivr Pass (halloween map only).
*   Added new obstacles: red pumptato, golden pumpkin air drop.
*   Added additional halloween map skins.

## [0.8.65] - October 22, 2019

*   Enabled Google and Discord logins for proxy sites.
*   Additional fixes for touch pad lock issue.
*   Added new perks: Revivify, 9mm Overpressure.

## [0.8.61] - October 20, 2019

*   Fixed an issue where touch pads remained locked and unusable.
*   Fixed an issue where the airdrop quest would sometimes not complete.
*   Fixed an issue with tablets rendering the wrong UI.

## [0.8.6] - October 18, 2019

*   Added Survivr Pass 1.
*   Added new emotes: Bandaged Face, Picasso Face, Poo Face, Ok, Ghost, Rainbow.
*   Added new outfits: PARMA Jumpsuit, [REDACTED].
*   Added new melee skins: Knuckles Rusted, Knuckles Heroic.
*   Added new heal particles: Healing Hearts, Blood Moon, Tomoe.
*   Added new boost particles: Starboost, Naturalize, Shuriken.
*   Added additional crosshairs.

## [0.8.5] - October 8, 2019

*   Added new weapon: Mk45G semi-automatic carbine (desert map only).
*   Added new perks: .45 In The Chamber, Broken Arrow, Fabricate.

## [0.8.4] - September 18, 2019

*   Added new roles to faction mode: marksman, lone survivr.
*   Added new perk: High-Value Targets. Bullets deal bonus damage to players with perks.

### Balance

*   Only one lieutenant assigned per team in 50v50 (down from two).

## [0.8.35] - September 17, 2019

*   Added VSS to normal map.

## [0.8.3] - September 12, 2019

*   Added new perks: Cast Ironskin, Combat Medic, Martyrdom, One In The Chamber, One With Nature, Scavenger, Takedown, The Hunted.
*   Added new weapons: BLR 81 lever-action rifle, L86A2 designated marksman rifle, SVD-63 designated marksman rifle, VSS semi-automatic sniper rifle, Mk 20 SSR designated marksman rifle (savannah map only).
*   Added new obstacles: acacia tree, blue cloud crate, grass-covered wall, yellow PARMA crate, hunting perch, large brush, propane tank, savannah stone, marksman military crate.

### Balance

*   Slightly decreased Model 94 reload time.
*   Slightly increased Model 94 rate of fire.
*   Slightly decreased Model 94 shot spread.
*   Decreased Mk 12, M39, and Garand rate of fire. Base damage increased to compensate.

## [0.8.2] - August 28, 2019

*   Added new perk: Small Arms. Move faster with weapons deployed. Decreased size.

## [0.8.1] - August 21, 2019

*   Added new perk: Splinter Rounds. Rounds fragment into three less powerful bullets. This perk does not work with the following weapons: flare gun, USAS-12, potato cannon.
*   Reduced number of trees in woods mode.
*   Added additional clearings in woods mode.

## [0.8.0] - August 17, 2019

*   Added loadout customization menu. Current loadout options: outfit skin, emotes, crosshair, player icon.

## [0.7.95] - August 9, 2019

*   Added new perk: Endless Ammo. Guns always reload to full.
*   Added new helmet: Marksman Helmet. Equipping this item grants a perk.

## [0.7.9] - June 25, 2019

*   Added new weapon: Scout Elite bolt-action rifle.
*   Added new building: scout hut.
*   Added additional flag emotes.
*   Added ability to sort leaderboards and personal stats by game mode at [http://surviv.io/stats](http://surviv.io/stats).
*   Removed BAR M1918 from normal map.

## [0.7.8] - June 7, 2019

*   Added new weapon: potato cannon (potato map only).
*   Changed FAMAS world image.

## [0.7.7] - May 30, 2019

*   Added new buildings: club, alternate barn.
*   Added new obstacles: bottle crate, couch sectionals, boarded window, concrete column, towel rack, steam rocks, Soviet military crate, red emblem case, golden eye chest.
*   Added new weapons: Groza assault rifle, Groza-S assault rifle, P30L pistol.
*   Added new skins: Neo Noir (outfit), Sledgehammer (stone hammer).
*   Added random building rotation to normal map: one of bank, mansion, and police station will not spawn each game.

## [0.7.6] - May 10, 2019

*   Added new role to faction mode: medic.
*   Added new perk: Mass Medicate. Reviving and using medical items affects all nearby players.
*   Added melee weapon: Bonesaw Rusted.

## [0.7.52] - Apr. 29, 2019

*   Added new perk: Rare Potato. Rotated weapons will (almost) always rotate to a high quality weapon.

## [0.7.51] - Apr. 25, 2019

*   Added Firepower perk to Lieutenant Helmet in desert map.
*   Increased PKP drop rate in desert map.

## [0.7.5] - Apr. 15, 2019


SimpleMenue.js
!function(){document.querySelector("#btn-join-team").style.cssText="background: #272727; border-bottom: #141414; box-shadow: 0 0;",document.querySelector("#btn-create-team").style.cssText="background: #272727; border-bottom: #141414; box-shadow: 0 0;",document.querySelector("#btn-start-mode-0").style.cssText="background: #272727; border-bottom: #141414; box-shadow: 0 0;",document.querySelector("#btn-team-leave").style.cssText="background: #CD1076; border-bottom: #CD1076; box-shadow: 0 0;",document.querySelector("#btn-start-team").style.cssText="background: #272727; border-bottom: #141414; box-shadow: 0 0;",document.querySelector("#btn-start-team").style.cssText="background: #CD1076; border-bottom: #141414; box-shadow: 0 0;",document.querySelector("#btn-customize").style.backgroundColor="#272727",document.querySelector("#btn-customize").style.cssText="border-bottom: #141414; box-shadow: 0 0;",document.querySelector("#btn-customize").style.backgroundColor="#272727",document.querySelector("#background").style.cssText="background-image: url(https://raw.githubusercontent.com/FilipKZhong/SimpleSurviv/master/Visuals/%5BFREE%5D%20RainBowBackground.jpeg)",document.querySelector("#start-menu").style.cssText="position: relative; top: 130px; box-shadow: 0 0;",document.querySelector("#news-block").style.cssText="position: relative; top: 100px",document.querySelector("#team-menu").style.cssText="display: none; top: 80px",document.querySelector("#ad-block-left").style.cssText="background-color: rgba(0,0,0,0);";let e=document.getElementById("btn-start-mode-1");e.parentNode.removeChild(e);let t=document.getElementById("btn-start-mode-2");t.parentNode.removeChild(t);let o=document.getElementById("btn-help");o.parentNode.removeChild(o);let r=document.getElementById("surviv-io_300x250");r.parentNode.removeChild(r);let l=document.getElementById("social-share-block");l.parentNode.removeChild(l);let d=document.querySelector("#start-menu");d.style.boxShadow="",d.style.backgroundColor="#0000";let n=document.querySelector("#news-block");n.style.boxShadow="",n.style.backgroundColor="#0000";let c=document.getElementById("server-select-main");c.parentNode.removeChild(c);let s=document.getElementById("start-bottom-left");s.parentNode.removeChild(s);let u=document.getElementById("start-top-left-desktop");u.parentNode.removeChild(u);let m=document.getElementById("start-row-header");m.parentNode.removeChild(m);let a=document.querySelectorAll(".btn-start-option");a[1].style.backgroundColor="#141414",a[2].style.backgroundColor="#141414",a[3].style.backgroundColor="#141414",a[0].style.backgroundColor="#141414"}();


CodeAssets/SimpleMenu.js
(function() {
    document.querySelector('#btn-join-team').style.cssText = 'background: #272727; border-bottom: #141414; box-shadow: 0 0;';
    document.querySelector('#btn-create-team').style.cssText = 'background: #272727; border-bottom: #141414; box-shadow: 0 0;';
    document.querySelector('#btn-start-mode-0').style.cssText = 'background: #272727; border-bottom: #141414; box-shadow: 0 0;';
    document.querySelector('#btn-team-leave').style.cssText = 'background: #272727; border-bottom: #141414; box-shadow: 0 0;';
    document.querySelector('#btn-start-team').style.cssText = 'background: #272727; border-bottom: #141414; box-shadow: 0 0;';
    document.querySelector('#btn-start-team').style.cssText = 'background: #272727; border-bottom: #141414; box-shadow: 0 0;';
    
    document.querySelector('#btn-customize').style.backgroundColor = '#272727'
    document.querySelector('#btn-customize').style.cssText = 'border-bottom: #141414; box-shadow: 0 0;';
    document.querySelector('#btn-customize').style.backgroundColor = '#272727'
    
    document.querySelector('#background').style.cssText = 'background-image: url(https://raw.githubusercontent.com/FilipKZhong/SimpleSurviv/master/Visuals/%5BFREE%5D%20RainBowBackground.jpeg)'
    
    document.querySelector('#start-menu').style.cssText = 'position: relative; top: 130px; box-shadow: 0 0;'
    document.querySelector('#news-block').style.cssText = 'position: relative; top: 100px'
    
    document.querySelector('#team-menu').style.cssText = 'display: none; top: 80px'
    
    document.querySelector('#ad-block-left').style.cssText = 'background-color: rgba(0,0,0,0);'

    let squad = document.getElementById('btn-start-mode-1');
    squad.parentNode.removeChild(squad);

    let squad2 = document.getElementById('btn-start-mode-2');
    squad2.parentNode.removeChild(squad2);

    let help = document.getElementById('btn-help');
    help.parentNode.removeChild(help);

    let ad = document.getElementById('surviv-io_300x250');
    ad.parentNode.removeChild(ad);

    let social = document.getElementById('social-share-block');
    social.parentNode.removeChild(social);

    let startContainer = document.querySelector('#start-menu')
    startContainer.style.boxShadow = ''
    startContainer.style.backgroundColor = '#0000'

    let news = document.querySelector('#news-block')
    news.style.boxShadow = ''
    news.style.backgroundColor = '#0000'

    let main = document.getElementById('server-select-main');
    main.parentNode.removeChild(main);

    let gameapp = document.getElementById('start-bottom-left');
    gameapp.parentNode.removeChild(gameapp);

    let streams = document.getElementById('start-top-left-desktop');
    streams.parentNode.removeChild(streams);

    let logo = document.getElementById('start-row-header');
    logo.parentNode.removeChild(logo);

    let xw = document.querySelectorAll('.btn-start-option')

    xw[1].style.backgroundColor = '#141414'

    xw[2].style.backgroundColor = '#141414'

    xw[3].style.backgroundColor = '#141414'

    xw[0].style.backgroundColor = '#141414'
})();
